src/chainparamsbase.cpp

```diff
@@ -91,9 +92,10 @@ std::string ChainNameFromCommandLine()
 {
     bool fRegTest = gArgs.GetBoolArg("-regtest", false);
     bool fTestNet = gArgs.GetBoolArg("-testnet", false);
+    bool fMainNet = gArgs.GetBoolArg("-mainnet", false);
 
-    if (fTestNet && fRegTest)
-        throw std::runtime_error("Invalid combination of -regtest and -testnet.");
+    if (fTestNet + fRegTest + fMainNet >= 2)
```

Comment:
Nit missing braces {} according to developer notes

Reply:
Fixed for all `if` blocks in this method.